qiita_db/support_files/patches/74.sql

```diff
@@ -0,0 +1,3 @@
+-- Apr 2nd, 2019
+-- Add a new filepath type
+INSERT INTO qiita.filepath_type (filepath_type) VALUES ('qza');
```

Comment:
Can you comment on the meaning of qza?

Reply:
I thought it was common knowledge: qiime zip archive ...

Reply:
Gotcha. I'll make a note of it.